src/content/docs/filters/reference.mdx

```diff
@@ -0,0 +1,341 @@
+| `http.request.uri.path`  | `string`      | `"/quick-start"`                              |
+| `ip.src.accuracy_radius` | `string`      | `"2"`                                         |
+| `ip.src.asnum.country`   | `string`      | `"US"`                                        |
+| `ip.src.asnum.domain`    | `string`      | `"google.com"`                                |
+| `ip.src.asnum.name`      | `string`      | `"Google, LLC."`                              |
+| `ip.src.asnum.type`      | `string`      | `"business"`                                  |
+| `ip.src.asnum`           | `string`      | `"15169"`                                     |
+| `ip.src.city`            | `string`      | `"Mountain View"`                             |
+| `ip.src.continent.name`  | `string`      | `"North America"`                             |
+| `ip.src.continent`       | `string`      | `"NA"`                                        |
+| `ip.src.country.name`    | `string`      | `"United States"`                             |
+| `ip.src.country`         | `string`      | `"US"`                                        |
+| `ip.src.crawler.name`    | `string`      | `""`                                          |
+| `ip.src.crawler`         | `boolean`     | `false`                                       |
+| `ip.src.hosting`         | `boolean`     | `false`                                       |
+| `ip.src.lat`             | `string`      | `"37.33939"`                                  |
+| `ip.src.lon`             | `string`      | `"-121.89496"`                                |
+| `ip.src.mobile`          | `boolean`     | `false`                                       |
+| `ip.src.postal_code`     | `string`      | `"94043"`                                     |
+| `ip.src.proxy`           | `boolean`     | `false`                                       |
+| `ip.src.region`          | `string`      | `"California"`                                |
+| `ip.src.relay`           | `boolean`     | `false`                                       |
+| `ip.src.service`         | `string`      | `""`                                          |
+| `ip.src.timezone.name`   | `string`      | `"America/Los_Angeles"`                       |
+| `ip.src.tor`             | `boolean`     | `false`                                       |
+| `ip.src.vpn`             | `boolean`     | `false`                                       |
+| `ip.src`                 | `Ip`          | `8.8.8.8`                                     |
+
+All `ip.src.*` fields reflect information about the client IP address (the
+request source IP). These fields require data from our IP reputation database so
+a call to the Arcjet API is required. Other fields are analyzed locally.
+
+Arcjet only ever makes a single call to our Cloud API regardless of the rule
+configuration. The Arcjet API has been designed for high performance and low
```

Comment:
```suggestion
configuration. Our Cloud API has been designed for high performance and low
```

Reply:
We'll change this in a different PR to make it consistent across the docs.